src/reducers/index.js

```diff
@@ -9,6 +9,7 @@ import viewHandler from './viewHandler';
 import filters from './filterHandler';
 import table from './tableHandler';
 import commentsPanel from './commentsPanel';
+import tables from './tables';
```

Comment:
can you pls with the refactoring move the logic from the tableHandler (L: 10 above to move to  tables) ?  Or should we use 'table' for the entities ?!? The idea is that we should agree on a form. Either plural or singular and then have consistency across the app.